html-templates/simple.mustache

```diff
@@ -319,7 +320,7 @@
 							<th>End Column</th>
 						</tr>
 					</thead>
-					<tbody/>
+					<tbody></tbody>
```

Comment:
Fixes w3 validation error:
`Self-closing syntax (/>) used on a non-void HTML element. Ignoring the slash and treating as a start tag.`